packages/toolpad-app/src/toolpad/AppEditor/AppEditorShell.tsx

```diff
@@ -153,12 +153,12 @@ export default function AppEditorShell({ appId, ...props }: ToolpadShellProps) {
     <ToolpadShell
       actions={
         <Stack direction="row" gap={1} alignItems="center">
-          <IconButton onClick={domApi.undo}>
```

Comment:
We try to avoid implementation details in the tests and also using things like test ids unless strictly necessary, which is also according to Playwright's recommendations of prioritizing user-facing attributes https://playwright.dev/docs/selectors#best-practices.
in this case I guess we could use the tooltip titles for the undo/redo buttons, so test ids should not be needed?